server/controllers/auth.js

```diff
@@ -84,12 +84,48 @@ exports.loginUser = asyncHandler(async (req, res, next) => {
   }
 });
 
+// @route POST /auth/demo-login
+// @desc Login demo user
+// @access Public
+exports.demoLogin = asyncHandler(async (req, res, next) => {
+
+  const { email, password } = req.body;
+
+  if (!email || !password || email !=="demo@kanban.com" || password !== "123456") {
+    res.sendStatus(400)
```

Comment:
A `401` will be more suitable for this. But you may use a `400` if you are considering only that either none or some of the required fields are missing.